contracts/protocol/facets/ConfigHandlerFacet.sol

```diff
@@ -427,6 +427,8 @@ contract ConfigHandlerFacet is IBosonConfigHandler, ProtocolBase {
     function setMinResolutionPeriod(uint256 _minResolutionPeriod) public override onlyRole(ADMIN) nonReentrant {
         // Make sure _maxResolutionPeriod is greater than 0
         checkNonZero(_minResolutionPeriod);
+        // Make sure _minResolutionPeriod is less than _maxResolutionPeriod
+        require(_minResolutionPeriod < protocolLimits().maxResolutionPeriod, INVALID_RESOLUTION_PERIOD);
```

Comment:
I suggest caching `protocolLimits()`. 

Also in `setMaxResolutionPeriod`

Reply:
Also, I think we could use `<=` instead of strict inequality. I don't see a logical fault in it and I checked that it would not break the offer creation.